src/modules/configurator/components/saved-configurations/Configurations.tsx

```diff
@@ -0,0 +1,19 @@
+import ConfigureLinkButton from "../ConfigureLinkButton";
+import ConfigurationsEmpty from "./ConfigurationsEmpty";
+import { useRecoilState } from "recoil";
+import { userConfigurationsState } from "../../state/configurationsState";
+import ConfigurationsSaved from "./ConfigurationsSaved";
+
+export default function Configurations() {
+  const [userConfigurations] = useRecoilState(userConfigurationsState);
```

Comment:
Use useRecoilValue here.

Reply:
Switched useRecoilState for useRecoilValue.